test/groovy/TransportRequestUploadFileTest.groovy

```diff
@@ -1,5 +1,8 @@
+import static org.hamcrest.Matchers.*
```

Comment:
If you add this you should not need the other Matchers imports below, right?

Reply:
additional imports removed